Cargo.toml

```diff
@@ -81,6 +82,7 @@ evm-debug = ["ethcore/evm-debug"]
 evm-debug-tests = ["ethcore/evm-debug-tests"]
 slow-blocks = ["ethcore/slow-blocks"]
 final = ["ethcore-util/final"]
+sstore = ["ethcore-secstore"]
```

Comment:
Maybe it will be better to enable it later? When this functionality will be finished && polished. Now it has only basic features && it is not usable at all. I don't mind fixing any compilation issues by myself until it is ready for production, if you're worried about it.